pkg/controller/bundles/bundles_syncer.go

```diff
@@ -87,6 +95,26 @@ func (syncer *LeafHubBundlesSpecSync) Start(stopChannel <-chan struct{}) error {
 	return nil
 }
 
+func readK8sPoolSizeEnvVar(log logr.Logger) int {
+	envK8sClientsPoolSize, found := os.LookupEnv(envVarK8sClientsPoolSize)
+	if !found {
+		log.Info("readK8sPoolSizeEnvVar", fmt.Sprintf("env variable %s not found", envVarK8sClientsPoolSize),
+			fmt.Sprintf("using default value %d", defaultK8sClientsPoolSize))
+
+		return defaultK8sClientsPoolSize
+	}
+
+	value, err := strconv.Atoi(envK8sClientsPoolSize)
+	if err != nil {
+		log.Info("readK8sPoolSizeEnvVar", fmt.Sprintf("failed to convert variable %s, value: %s",
```

Comment:
no need to log function name, no need for double use of fmt.Sprintf.
need to add another check for illegal value (added <1 check)
```
value, err := strconv.Atoi(envK8sClientsPoolSize)
if err != nil || value < 1 {
	log.Info(fmt.Sprintf("env var %s invalid value: %s, using default value %d", envVarK8sClientsPoolSize,
		envK8sClientsPoolSize, defaultK8sClientsPoolSize))
```

Reply:
done